milestones.go

```diff
@@ -212,3 +212,34 @@ func (s *MilestonesService) GetMilestoneIssues(pid interface{}, milestone int, o
 
 	return i, resp, err
 }
+
+// GetMilestoneMergeRequestsOptions represents the available GetMilestoneMergeRequests() options.
```

Comment:
Please keep comments to ~80 chars when possible (so links are sometimes longer, but that's OK)

Reply:
Fixed